tests/test_amclient.py

```diff
@@ -69,9 +70,9 @@ def test_close_completed_transfers_transfers(self):
         """Test closing completed transfers when there are completed transfers
         to close.
         """
-        response = amclient.AMClient(
-            am_api_key=AM_API_KEY, am_user_name=AM_USER_NAME,
-            am_url=AM_URL).close_completed_transfers()
+        response = amclient.AMClient(am_api_key=AM_API_KEY,
+                                     am_user_name=AM_USER_NAME,
+                                     am_url=AM_URL).close_completed_transfers()
```

Comment:
I have one more suggestion inspired by Django's subclass of `TestCase`. Should we have a `tearUp` method that initializes the client in `self.client`? It would remove a bunch of duplicated code.

    resp = self.client.close_completed_transfers)

Or in cases where you have to change some of the properties...

    self.client.transfer_uuid = "63fcc1b0-f83d-47e6-ac9d-a8f8d1fc2ab9"
    resp = self.client.get_transfer_status()

Reply:
@sevein I like this as a future enhancement but would be more comfortable making a new issue of this so that I can merge this work for the client waiting on it and to consider the possible solutions. Does this sound right? If `self.client.transfer_uuid` isn't set in a tear-up method when the class is initialised then `setattr(object, name, value)` needs to be called to set one after the fact - we can't just access the members if they don't exist?